Keen.NetStandard/Keen.NetStandard.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
I chose it because it's the latest stable. 9.0.1 also supports netstandard. Should I move back to 9.0.1?